pkg/v1/tkg/test/tkgctl/tkgs/tkgs_workload_cluster_test.go

```diff
@@ -40,7 +40,7 @@ var _ = Describe("TKGS - Create workload cluster use cases", func() {
 			})
 			When("cluster class cli feature flag (features.global.package-based-lcm-beta) is set to true", func() {
 				BeforeEach(func() {
-					//set the cli feature flag as true -  (features.global.package-based-lcm-beta)
+					// set the cli feature flag as true -  (features.global.package-based-lcm-beta)
```

Comment:
this is obvious from the code. and as you see, carrying the comment just makes more stuff to mantain in the future. Maybe take the comment out all together